<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="main">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>768</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>font.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="cont">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>450</y>
     <width>88</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>cont.png</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="line1">
   <property name="geometry">
    <rect>
     <x>555</x>
     <y>335</y>
     <width>255</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 255px;
height: 40px;
left: 555px;
top: 335px;

background: rgba(255, 255, 255, 0.2);
border-radius: 3px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line2">
   <property name="geometry">
    <rect>
     <x>555</x>
     <y>392</y>
     <width>255</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 255px;
height: 40px;
left: 555px;
top: 335px;

background: rgba(255, 255, 255, 0.2);
border-radius: 3px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="arrow">
   <property name="geometry">
    <rect>
     <x>869</x>
     <y>375</y>
     <width>53</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Arrow 1.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="people">
   <property name="geometry">
    <rect>
     <x>692</x>
     <y>229</y>
     <width>193</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>people.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="addp">
   <property name="geometry">
    <rect>
     <x>694</x>
     <y>273</y>
     <width>191</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>people2.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="text">
   <property name="geometry">
    <rect>
     <x>808</x>
     <y>289</y>
     <width>7</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Source Sans Pro;
font-style: normal;
font-weight: 600;
font-size: 10px;
line-height: 15px;
/* identical to box height */
color: #FFFFFF;
</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="add">
   <property name="geometry">
    <rect>
     <x>819</x>
     <y>292</y>
     <width>10</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>add (1) 1.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="sub">
   <property name="geometry">
    <rect>
     <x>796</x>
     <y>292</y>
     <width>10</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>minus (1) 1.png</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="cityLine">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>229</y>
     <width>380</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 380px;
height: 45px;
left: 128px;
top: 228.9px;

background: #FFFFFF;
border: 2px solid #197EC2;
box-sizing: border-box;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataLine">
   <property name="geometry">
    <rect>
     <x>506</x>
     <y>229</y>
     <width>190</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 380px;
height: 45px;
left: 128px;
top: 228.9px;

background: #FFFFFF;
border: 2px solid #197EC2;
box-sizing: border-box;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search">
   <property name="geometry">
    <rect>
     <x>1048</x>
     <y>229</y>
     <width>190</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 190px;
height: 45px;
left: 1048px;
top: 226.9px;

background: #197EC2;
border: 2px solid #197EC2;
box-sizing: border-box;

font-family: Source Sans Pro;
font-style: normal;
font-weight: bold;
font-size: 12px;
line-height: 18px;
/* identical to box height */

display: flex;
align-items: flex-end;
text-align: center;
text-transform: uppercase;

color: #FFFFFF;
</string>
   </property>
   <property name="text">
    <string>ПОИСК</string>
   </property>
  </widget>
  <widget class="QLabel" name="back">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>20</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>back.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="main_label">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>65</y>
     <width>1100</width>
     <height>698</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>main_label.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="city">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>206</y>
     <width>144</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>city.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="date">
   <property name="geometry">
    <rect>
     <x>508</x>
     <y>206</y>
     <width>103</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>date.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="person">
   <property name="geometry">
    <rect>
     <x>698</x>
     <y>206</y>
     <width>125</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>user.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="chikago">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>323</y>
     <width>635</width>
     <height>199</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Чикаго.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="profile">
   <property name="geometry">
    <rect>
     <x>1078</x>
     <y>15</y>
     <width>160</width>
     <height>84</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>profile.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="photo">
   <property name="geometry">
    <rect>
     <x>1105</x>
     <y>23</y>
     <width>23</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>prof.photo.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="exit">
   <property name="geometry">
    <rect>
     <x>1134</x>
     <y>75</y>
     <width>48</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>exit.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="hide">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>36</y>
     <width>12</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>arrow_uo.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="hotel">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>236</y>
     <width>350</width>
     <height>404</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>hotel.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="comment">
   <property name="geometry">
    <rect>
     <x>413</x>
     <y>477</y>
     <width>730</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 730px;
height: 55px;
left: 413px;
top: 477px;

font-family: Source Sans Pro;
font-style: normal;
font-weight: 600;
font-size: 11px;
line-height: 14px;
display: flex;
align-items: flex-end;

color: #FFFFFF;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="com">
   <property name="geometry">
    <rect>
     <x>222</x>
     <y>572</y>
     <width>860</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 860px;
height: 75px;
left: 222px;
top: 572px;

background: #FFFFFF;
border: 4px solid #197EC2;
box-sizing: border-box;

position: absolute;
width: 730px;
height: 55px;
left: 413px;
top: 477px;

font-family: Source Sans Pro;
font-style: normal;
font-weight: 600;
font-size: 11px;
line-height: 14px;
display: flex;
align-items: flex-end;

color: #000000;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="send">
   <property name="geometry">
    <rect>
     <x>1082</x>
     <y>572</y>
     <width>60</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>butt_send.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="bron">
   <property name="geometry">
    <rect>
     <x>637</x>
     <y>405</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Забронировать.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="rew">
   <property name="geometry">
    <rect>
     <x>846</x>
     <y>405</y>
     <width>46</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Отзывы1.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="descr">
   <property name="geometry">
    <rect>
     <x>464</x>
     <y>405</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Описание.png</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="dateLINE">
   <property name="geometry">
    <rect>
     <x>511</x>
     <y>506</y>
     <width>137</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 137px;
height: 35px;

background: #FFFFFF;
border: 2px solid #197EC2;
box-sizing: border-box;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dateLine">
   <property name="geometry">
    <rect>
     <x>802</x>
     <y>506</y>
     <width>137</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 137px;
height: 35px;

background: #FFFFFF;
border: 2px solid #197EC2;
box-sizing: border-box;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BronButton">
   <property name="geometry">
    <rect>
     <x>603</x>
     <y>587</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 190px;
height: 45px;
left: 1048px;
top: 226.9px;

background: #197EC2;
border: 2px solid #197EC2;
box-sizing: border-box;

font-family: Source Sans Pro;
font-style: normal;
font-weight: 600;
font-size: 13px;
line-height: 16px;
display: flex;
align-items: flex-end;

color: #FFFFFF;
/* identical to box height */

</string>
   </property>
   <property name="text">
    <string>Забронировать</string>
   </property>
  </widget>
  <widget class="QLabel" name="Brontext">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>450</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Source Sans Pro;
font-style: normal;
font-weight: 600;
font-size: 13px;
line-height: 16px;
display: flex;
align-items: flex-end;

color: #000000;</string>
   </property>
   <property name="text">
    <string>Забронировано с</string>
   </property>
  </widget>
  <widget class="QLabel" name="Brontext_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>480</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Source Sans Pro;
font-style: normal;
font-weight: 600;
font-size: 13px;
line-height: 16px;
display: flex;
align-items: flex-end;

color: #00000;</string>
   </property>
   <property name="text">
    <string>по</string>
   </property>
  </widget>
  <widget class="QLabel" name="showIN">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>450</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Source Sans Pro;
font-style: normal;
font-weight: 600;
font-size: 13px;
line-height: 16px;
display: flex;
align-items: flex-end;

color: #000000;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="showOUT">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>480</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Source Sans Pro;
font-style: normal;
font-weight: 600;
font-size: 13px;
line-height: 16px;
display: flex;
align-items: flex-end;

color: #000000;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="new_click">
   <property name="geometry">
    <rect>
     <x>793</x>
     <y>312</y>
     <width>431</width>
     <height>84</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Нью - Йорк.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="Berlin_click">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>540</y>
     <width>445</width>
     <height>198</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Берлин.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="moscow_click">
   <property name="geometry">
    <rect>
     <x>792</x>
     <y>428</y>
     <width>435</width>
     <height>84</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Москва.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="london_click">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>540</y>
     <width>611</width>
     <height>199</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Лондон.jpg</pixmap>
   </property>
  </widget>
  <zorder>main</zorder>
  <zorder>main_label</zorder>
  <zorder>cont</zorder>
  <zorder>line1</zorder>
  <zorder>line2</zorder>
  <zorder>arrow</zorder>
  <zorder>people</zorder>
  <zorder>addp</zorder>
  <zorder>text</zorder>
  <zorder>cityLine</zorder>
  <zorder>dataLine</zorder>
  <zorder>search</zorder>
  <zorder>back</zorder>
  <zorder>city</zorder>
  <zorder>date</zorder>
  <zorder>person</zorder>
  <zorder>add</zorder>
  <zorder>sub</zorder>
  <zorder>chikago</zorder>
  <zorder>profile</zorder>
  <zorder>photo</zorder>
  <zorder>exit</zorder>
  <zorder>hotel</zorder>
  <zorder>comment</zorder>
  <zorder>com</zorder>
  <zorder>send</zorder>
  <zorder>bron</zorder>
  <zorder>rew</zorder>
  <zorder>descr</zorder>
  <zorder>hide</zorder>
  <zorder>dateLINE</zorder>
  <zorder>dateLine</zorder>
  <zorder>BronButton</zorder>
  <zorder>Brontext</zorder>
  <zorder>Brontext_2</zorder>
  <zorder>showIN</zorder>
  <zorder>showOUT</zorder>
  <zorder>new_click</zorder>
  <zorder>Berlin_click</zorder>
  <zorder>moscow_click</zorder>
  <zorder>london_click</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
